plugin/trino-opa/src/main/java/io/trino/plugin/opa/OpaAccessControl.java

```diff
@@ -90,6 +90,7 @@ public sealed class OpaAccessControl
     private final LifeCycleManager lifeCycleManager;
     private final OpaHighLevelClient opaHighLevelClient;
     private final boolean allowPermissionManagementOperations;
+    protected final boolean includeUserPrincipal;
```

Comment:
Personally I'd prefer if this could be made private and the create an access method.